.github/ISSUE_TEMPLATE/proposal.yml

```diff
@@ -0,0 +1,31 @@
+name: Proposal
+description: Proposal for feature to include in Interop 2023
```

Comment:
I wonder if it makes sense to have a separate issue template for investigation proposals, asking for more things like "what is the proposed outcome of this investigation effort, what is being investigated, etc"

Reply:
Yes, we should do that!